src/main/java/seedu/storemando/model/item/Item.java

```diff
@@ -28,7 +28,7 @@
      * Every field must be present and not null.
      */
     public Item(ItemName name, Quantity quantity, ExpiryDate expiryDate, Location location, Set<Tag> tags) {
-        requireAllNonNull(name, quantity, location, tags);
+        requireAllNonNull(name, quantity, expiryDate, location, tags);
```

Comment:
understand that for this file the sequence in which what is placed first does not matter, but for readability and consistency of our code base we should perhaps standardise it, following the google docs sequence